app/scripts/modules/amazon/src/loadBalancer/configure/classic/AdvancedSettings.tsx

```diff
@@ -23,44 +23,43 @@ class AdvancedSettingsImpl extends React.Component<IAdvancedSettingsProps> {
         <FormikFormField
           name="healthTimeout"
           label="Timeout"
+          required={true}
+          fastField={false} /* This field depends on  healthInterval */
```

Comment:
If "depends on other field" is the test, would it make sense to rename this prop to reflect that? Would just make it more obvious.

Reply:
`derived` is accurate but a little opaque. `dependsOnOtherField` seems ok.

It's unfortunate that we have to push this cognitive overhead up to form code, but a boolean is a lot better than needing to explicitly define field dependencies so I'll take it.

Reply:
re: `dependsOnField="healthInterval` in my experience adding dummy config rarely buys as much future value as one anticipates, requirements change and the dummy config eventually has to keep up or die. It may also get confusing when we run into the first case of a field depending on multiple other fields (or some kind of relationship that goes beyond the concept of specific field dependencies). Do we add some additional no-op array syntax to the mix then?

There's something to be said for not abstracting over Formik nomenclature — I wouldn't be upset about it staying as `fastField`.

Reply:
It's funny because I basically had this whole conversation in my head and settled on `fastField` before I submitted this PR.   Keeping as is then.